cmd/tailscale/cli/up.go

```diff
@@ -294,6 +296,29 @@ func runUp(ctx context.Context, args []string) error {
 		},
 	}
 
+	if upArgs.json {
+		opts.Notify = func(n ipn.Notify) {
+			if n.ErrMessage != nil {
+				fmt.Fprintf(os.Stdout, `{"error":%q}\n`, *n.ErrMessage)
```

Comment:
Any reason you're using fmt.Fprintf instead of encoding/json?

Reply:
I find it easier to reason about fmt for a string than the JSON package.